dsgrid/utils/spark.py

```diff
@@ -108,8 +146,8 @@ def read_dataframe(filename, cache=False, require_unique=None, read_with_spark=T
     ----------
     filename : str | Path
         path to file
-    cache : bool
-        If True, cache the DataFrame in memory.
+    table_name : str | None
+        If set, cache the DataFrame in memory. Must be unique.
```

Comment:
read the cached DataFrame from memory?

Reply:
This function will always read from the file on disk. The table_name can set to read and then cache. Presumably, the user will use that name rather than calling this function again.